content/polkadot/async-backing-shallow/slides.md

```diff
@@ -22,12 +22,19 @@ Lets get to it
 
 - Synchronous vs asynchronous
 - Why is asynchronous backing desirable?
+
 <!-- .element: class="fragment" data-fragment-index="1" -->
+
 - High level mechanisms of async backing
+
 <!-- .element: class="fragment" data-fragment-index="2" -->
+
 - The unincluded segment, and prospective parachains
```

Comment:
dprint want us to use a full space, that is OK for reveal, but not ergonomic to use. Better story on `fragment`s use in this book should be done :thinking: - that is a reveal-md problem more than ours though.